react/react1/week3/todolist-fetch/src/components/TodoItem.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+import "../App.css";
+import FancyBorder from "./FancyBorder";
+
+export function TodoItem({ description, deadline, id, deleteTodo, editTodo }) {
+  const [checked, setChecked] = useState("unchecked");
```

Comment:
An alternative way to go here, would be to add a 'checked' property to the todo object like you did in your previous week2-hw. That way you don't have to maintain a checked variable here outside of your todo object
When you first fetch the todos you could do this to add a property:
```
 useEffect(() => {
    fetch(
      "https://gist.githubusercontent.com/benna100/391eee7a119b50bd2c5960ab51622532/raw"
    )
      .then((response) => response.json())
      .then((data) =>  setTodos(data.map((item) => {
                return { ...item, checked: false }
            })));
  }, []);
```